### ISSUE TITLE
Add binary implementation

### ISSUE THREAD

**Issue by robinvdvleuten** [robinvdvleuten/php-ulid#2]
See [ramsey/uuid](https://github.com/ramsey/uuid) for a similar implementation.

**Comment by lainosantos** [comment]
Hello,

any news for this?

Thanks.

**Comment by robinvdvleuten** [comment]
Nope. As I am using Postgres as datastore, there is no need for me to have a binary implementation. So be my guest to send a PR for this if you need it yourself.
